### PR TITLE
Finally fix spelling mistakes in de language

### PR DESCRIPTION
**Pull request by dezem**
Finally fix spelling mistakes in de language.
Only one commit.

### CODE REVIEW

File: romfs/lang/de.json [hunk 1]
```diff
@@ -25,15 +25,15 @@
         },
         "applet": {
             "title": "Applet-Modus wird nicht unterstützt",
-            "desc": "Es kann zu Problemen bei der Verwendung des Awoo-Installers im Applet-Modus kommen.\nWenn du Probleme hast, wechsele bitte dazu über, den Awoo Installer über einen installierten Titel zu starten (halte R gedrückt, während du ein Spiel startest)!"
+            "desc": "Es kann zu Problemen bei der Verwendung des Awoo Installer's im Applet-Modus kommen.\nWenn du Probleme hast, starte den Awoo Installer über einen installierten Titel (halte R gedrückt, während du ein Spiel startest)!"
```
Comment (MegaPramuk) [inline romfs/lang/de.json]: "Installer's" is wrong.
German doesn't have the possessive apostrophe like English does.
Also while not really wrong, I'd change the sentence to `Es kann zu Problemen bei der Verwendung von Awoo-Installer im Applet-Modus kommen.` as that sounds more natural.

File: romfs/lang/de.json [hunk 2]
```diff
@@ -25,15 +25,15 @@
...
+                "desc": "Dateien können von anderen Geräten aus mit Tools wie NS-USBloader im\nTinfoil-Modus installiert werden. Um Dateien an deine Switch zu senden, öffne\neins dieser Programme auf deinem PC oder mobilen Gerät. Gebe die IP-Adresse\ndeiner Switches ein (auf dem Bildschirm aufgelistet), wähle die Dateien\naus und lade sie dann auf die Konsole hoch! Wenn die von dir verwendete\nSoftware dir nicht erlaubt, bestimmte Dateitypen auszuwählen.\nVersuche die Dateiendung in etwas umzubenennen, dass sie akzeptiert. Der Awoo Installer\nprüft keine Dateiendungen bei Netzinstallationen!\n\nFalls dass nicht klappt, kopiere einfach deine Dateien auf die SD-Karte\nund versuche die Option \"Von SD-Karte installieren\" im Hauptmenü!"
```
Comment (MegaPramuk) [inline romfs/lang/de.json]: `Falls dass nicht klappt` -> `Falls das nicht klappt`
If you cannot swap out dass with jenes, then you have to write it with only one s.

File: romfs/lang/de.json [hunk 3]
```diff
@@ -61,19 +61,19 @@
         "sd": {
             "help": {
                 "title": "Hilfe",
-                "desc": "Kopiere NSP, NSZ, XCI, or XCZ Dateien auf deine SD Karte,\nstöbere nach ihnen und wähle die zu installierenden Titel aus und drücke dann \ue0b3."
+                "desc": "Kopiere NSP, NSZ, XCI, oder XCZ Dateien auf deine SD Karte,\nsuche nach ihnen und wähle die zu installierenden Titel aus und drücke dann \ue0b3."
```
Comment (MegaPramuk) [inline romfs/lang/de.json]: `Kopiere NSP, NSZ, XCI, oder XCZ Dateien` -> `Kopiere NSP, NSZ, XCI oder XCZ Dateien`
In German you don't use a comma before und or oder (or similar words).

File: romfs/lang/de.json [hunk 4]
```diff
@@ -98,46 +98,46 @@
...
             "desc1": " installiert!",
-            "downloading": "Läd ",
+            "downloading": "Lädt ",
             "at": " herunter von "
         },
         "nca_verify": {
-            "title": "Ungültige NCA-Signature erkannt!",
-            "desc": "Unsachgemäß signierte Software sollte nur von vertrauenswürdigen Quellen\ninstalliert werden. Dateien, die Cartridge-Repacks und DLC-Freischaltgeräte enthalten,\nwerden immer diese Warnung anzeigen. Sie können diese Prüfung in den Einstellungen des Awoo Installers deaktivieren.\nnSind Sie sicher, dass Sie die Installation fortsetzen möchten?",
+            "title": "Ungültige NCA-Signatur erkannt!",
+            "desc": "Unsachgemäß signierte Software sollte nur von vertrauenswürdigen Quellen\ninstalliert werden. Dateien von gepackten Spiele Modulen und DLC-Freischaltung enthalten,\nwerden immer diese Warnung anzeigen. Sie können diese Prüfung in den Einstellungen des Awoo Installer's deaktivieren.\nSind Sie sicher, dass Sie die Installation fortsetzen möchten?",
```
Comment (MegaPramuk) [inline romfs/lang/de.json]: Again, no possessive apostrophe.

File: romfs/lang/de.json [hunk 5]
```diff
@@ -98,46 +98,46 @@
...
+        "version_text": "Du hast die aktuellen Patches für die HOS Version installiert ",
         "title0": "Installiere Signatur Patches?",
-        "desc0": "Signatur Patches werden benötigt um offizielle Software zu installieren und zu spielen.",
-        "backup_failed": "Konnten Hekates patches.ini nicht sichern! Trotzdem installieren?",
-        "backup_failed_desc": "Falls du Heakte nicht nutzt kannst du diese Warnung ignorieren.",
-        "download_failed": "Konnten die Signatur Patches nicht herunterladen",
-        "download_failed_desc": "Eventuell sind die in den Einstellungen gestellte Quelle ungültig oder der Host ist offline.",
-        "version_text2": "Deine Signatur Patches wurden aktuallisiert für HOS Versionen bis ",
+        "desc0": "Signatur Patches werden benötigt, um offizielle Software zu installieren und zu spielen.",
+        "backup_failed": "Konnte Hekate's patches.ini nicht finden! Trotzdem installieren?",
```
Comment (MegaPramuk) [inline romfs/lang/de.json]: Possessive apostrophe.

File: romfs/lang/de.json [hunk 6]
```diff
@@ -98,46 +98,46 @@
...
+        "download_failed_desc": "Eventuell hast du eine ungültige Quelle in den Einstellung\nvom Awoo Installer eingestellt oder der Host ist derzeitig offline.",
+        "version_text2": "Deine Signatur Patches wurden aktualisiert, für deine HOS Version ",
         "install_complete": "Installation abgeschlossen!",
         "complete_desc": "Starte die Konsole neu um die Änderung zu übernehmen",
         "restart": "Neustarten",
         "later": "Mach ich später",
-        "extract_failed": "Konnten die Dateien nicht extrahieren!",
-        "restore_failed": "Konnten hekates originale patches.ini nicht wieder herstellen! Deinstallation fortsetzen?",
+        "extract_failed": "Konnte die Dateien nicht extrahieren!",
+        "restore_failed": "Konnte Hekate's originale patches.ini nicht wiederherstellen! Deinstallation fortsetzen?",
```
Comment (MegaPramuk) [inline romfs/lang/de.json]: Possessive apostrophe.

File: romfs/lang/de.json [hunk 7]
```diff
@@ -148,39 +148,39 @@
...
+            "end_desc": "Die Software wird nun beendet.",
+            "title_check_fail": "Keine Aktualisierungen gefunden",
             "desc_check_fail": "Du bist auf der aktuellsten Version des Awoo Installer!"
         },
         "credits": {
             "title": "Vielen Dank an die folgenden Leute!",
-            "desc": "- HookedBehemoth für NE MENGE Beiträge\n- Adubbz und andere Mitwirkende für Tinfoil\n- XorTroll für Plutonium und Goldleaf\n- blawar (Frauenwürger) und nicoboss für NSZ Unterstützung\n- Die netten Leute auf AtlasNX' Discuck (Immerhin ein paar davon)\n- Die netten Leute auf RetroNX' Discuck (ohne direkte Beteiligung)\n- namako8982 für die Momiji-Kunst\n- TheXzoron dafür das er ein baka ist"
+            "desc": "- HookedBehemoth für EINE MENGE Beiträge\n- Adubbz und andere Mitwirkende für Tinfoil\n- XorTroll für Plutonium und Goldleaf\n- blawar (Frauenwürger) und nicoboss für NSZ Unterstützung\n- Die netten Leute auf AtlasNX' Discuck (Immerhin ein paar davon)\n- Die netten Leute auf RetroNX' Discuck (ohne direkte Beteiligung)\n- namako8982 für die Momiji-Kunst\n- TheXzoron dafür das er ein baka ist"
```
Comment (MegaPramuk) [inline romfs/lang/de.json]: I'd rather change `EINE` to `eine` as German doesn't emphasises the same way English does.

File: romfs/lang/de.json (hunk 1 above)
Comment (dezem) [reply romfs/lang/de.json]: I can change it, and remove the apostrophe with the 's'
I found it optical better with the apostrophe 😉

File: romfs/lang/de.json (hunk 2 above)
Comment (dezem) [reply romfs/lang/de.json]: Oh a small typo, i will fix it 👍

File: romfs/lang/de.json (hunk 3 above)
Comment (dezem) [reply romfs/lang/de.json]: Fix it too

File: romfs/lang/de.json (hunk 7 above)
Comment (dezem) [reply romfs/lang/de.json]: I followed the english one, there is it also in upper cases written

File: romfs/lang/de.json [hunk 8]
```diff
@@ -98,46 +98,46 @@
...
+            "error": "Die gewünschte NCA ist nicht richtig signiert: "
         },
         "finished": [
-            "Geniese deine \"legalen Sicherheitskopien\"!",
-            "Ich bin mir sicher nachdem du das Spiel probiert hast wirst du Spaß dabei haben das spiel tatsächlich zu kaufen!",
-            "Dul halt das Spiel sichel gekauft? Nintendo-san dankel dil fül deinen Kauf!",
-            "DRM zu umgehen ist toll, stimmts?",
-            "Du hast wahrscheinlich sechs Bäume gerettet dadurch dass du das Spiel nicht gekauft hast! All das Plastik geht irgendwo hin!",
+            "Genieße deine \"legalen Sicherheitskopien\"!",
+            "Ich bin mir sicher, nachdem du das Spiel probiert hast. Wirst du Spaß dabei haben, das Spiel tatsächlich zu kaufen!",
```
Comment (HookedBehemoth) [inline romfs/lang/de.json]: eh?

File: romfs/lang/de.json [hunk 9]
```diff
@@ -148,39 +148,39 @@
...
             "language": "Sprache: ",
             "check_update": "Suche nach Updates für Awoo Installer",
-            "credits": "Ehrungen"
+            "credits": "Anerkennung"
         },
         "nca_warn": {
             "title": "Warnung!",
-            "desc": "Einige installierbare Dateien können bösartige Inhalte enthalten! Deaktiviere\ndiese Funktion nur, wenn du absolut sicher bist, dass die zu installierende\nSoftware vertrauenswürdig ist! Willst du die NCA-Signaturprüfung immer noch deaktivieren?",
-            "opt1": "Ja, ich möchte mein Gerät zerstören"
+            "desc": "Einige installierbare Dateien können bösartige Inhalte enthalten! Deaktiviere\ndiese Funktion nur, wenn du absolut sicher bist. Dass die zu installierende\nSoftware vertrauenswürdig ist! Willst du die NCA-Signaturprüfung immer noch deaktivieren?",
```
Comment (HookedBehemoth) [inline romfs/lang/de.json]: eh?²

File: romfs/lang/de.json (hunk 8 above)
Comment (dezem) [reply romfs/lang/de.json]: what do you want to say?

File: romfs/lang/de.json (hunk 9 above)
Comment (dezem) [reply romfs/lang/de.json]: again -> what do you want to say?

File: romfs/lang/de.json (hunk 8 above)
Comment (HookedBehemoth) [reply romfs/lang/de.json]: The sentence doesn't make any sense split that way.

Comment (dezem) [reply romfs/lang/de.json]: So keep the original? Only with upper "Spiel"
"Ich bin mir sicher nachdem du das Spiel probiert hast wirst du Spaß dabei haben das Spiel tatsächlich zu kaufen!"

File: romfs/lang/de.json (hunk 9 above)
Comment (dezem) [reply romfs/lang/de.json]: i reverte this one line 😄

**Comment by Huntereb** [comment]
I'll push out a 1.3.2 once this is resolved, just to get those new languages out. Don't want to leave this out of what's already a small release.

**Comment by dezem** [comment]
From my side all fixed. Waiting for @HookedBehemoth to accept.

**Comment by dezem** [comment]
@Huntereb one more task that I have to do. Or is everything okay so far?

**Pull request merged**